rodan/jobs/base.py

```diff
@@ -924,6 +927,41 @@ def run(self, runjob_id):
                     ]
                 )
 
+                # Update workflow run description with job info
+                wall_time = time.time() - start_time
+                snapshot_info = '\n\n{0}:\n    name: "{1}"\n    wall_time: "{2}"\n'.format(
```

Comment:
Put it all in double quotes. Either escape the double quotes or put the whole thing in a triple quote so you don't need to escape anything.